### ISSUE TITLE
Collaboration Cycle for OCTO Identity, Sign in page (and sign in modal), Sign in Transition

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#90672]
Please view the [Milestone](https://github.com/department-of-veterans-affairs/va.gov-team/milestone/1379) for all feedback and findings tickets associated with this Collaboration Cycle initiative. 
 
### VFS team name

Identity

### Product name

Sign in page (and sign in modal)

### Feature name

Sign in Transition

### GitHub label for team

identity

### GitHub label for product

identity-ux

### Public DSVA Slack channel for VFS team

[#secure-sign-in-transition](https://dsva.slack.com/archives/C068QM7CCKG)

## Kickoff questions
<details><summary>Toggle kickoff questions</summary>

### When did/will you start working on this product/feature?

Let's call it July 2024 (last month)

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Static pages

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

Yes

### Do you need to capture any additional analytics or metrics?

No

### Will a VA editor (Drupal) notice this change?

No

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity/login/usip-legacy-logins

### Verify all relevant materials provided to Governance Team

- [X] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Add the GitHub labels for your team, product, and feature to this ticket.

- [X] I acknowledge that I must add the GitHub labels for my team, product, and feature to this ticket.
Kickoff Slack Thread with VFS team: [Kickoff thread](https://dsva.slack.com/archives/C04S98PE8FK/p1723645000026699)
</details>

### Design Intent
<details>
  <summary>Toggle Design Intent instructions</summary>

##### VFS team actions
- [x] Review [Design Intent Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/design-intent) to understand what this touchpoint involves.
- [x] Schedule your Design Intent (with at least 2 business days lead time from now):
  - Open the [Calendly design intent calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
  - Select a date and time and click "Confirm"
  - Add your name and email
  - Click "Add Guests" and enter the email addresses for VFS team attendees
  - Click "Schedule Event"
- [x] Link all artifacts in the `Design Intent artifacts for review` section below at least two business days before the scheduled Design Intent. Please don't add artifacts in the comments section.
  
**Design Intent artifacts for review**

[See guidance on Design Intent artifacts.](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/design-intent#Designintent-Artifacts) Governance Team feedback is based on the artifacts provided here as well as information provided during the meeting. Please provide links to artifacts **at least two business days** before the scheduled meeting.

Required:
- [x] [Figma desktop user flow](https://www.figma.com/proto/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=65-42645&t=WoWY0BMR13Zksogx-1&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1)
- [x] [Figma mobile user flow](https://www.figma.com/proto/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=329-141598&t=sS2DIdQ2K3HMtQEz-0&scaling=min-zoom&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=65%3A42645)

Not required, but nice to have:
- [x] [Wireframes/prototype brainstorming (in Figma)](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=65-42645&t=BJjDDPzJjheVzv8z-0)

   - Note: our working plan is to move forward with Variation 2 which leads into Variation 2b as seen in the flow
   - Note: while these protos are for the [the sign in modal](https://www.va.gov/?next=loginModal), we'll essentially follow the same design for the [universal sign in page](https://www.va.gov/sign-in/?oauth=false)

Optional:
- [ ] Research plan
- [x] Any other artifacts you have so far: [intake ticket with CAIA](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90690) to work on wording that we'll be adding to the new legacy credential hideaway page
- [x] Questions we have for Gov team, that we'll want to discuss
   - Let's do another round of ye olde button vs link debate! The "create a new account" links. Keep as link (current) vs change to button, as shown [on USWDS](https://designsystem.digital.gov/templates/authentication-pages/multiple-sign-in-options/) (which, I know, is not VADS, but I hate to go through a collab cycle without talking about buttons and links at least once). 
   - Need more button q's? Of course we do! Our Login.gov button is currently red, and [outside of sanctioned use, which is blue](https://developers.login.gov/user-experience/sign-in-sign-out/) 😱  We also don't want to create a dark pattern by changing Login.gov button to blue right when we hide the blue buttons of DSL and MHV... or maybe we do so help direct folks to Login.gov? Maybe we use their white background option, but does that look too much like VADS secondary button? If we wait on that button color change, then how long? Also if there are other ideas for gracefully changing color on that? Maybe change it to purple first or outline in blue first to slowly transition from red to blue 🤷 
   - Wording placement for deprecation dates and info. Thinking about the balance of design vs accessibility. Do we continue that content below the sign in buttons as we have with our quick/emergency update about deprecation that currently shows on [the sign in modal](https://www.va.gov/?next=loginModal) and [sign in page](https://www.va.gov/sign-in/?oauth=false)? Move it up, move it down? move it a little to the left? oh! too much, back to the right.
      - Should we follow that same pattern on what I'm calling the deprecated credential hideaway page? 
  
##### Governance Team actions
- [x] [Design Intent Slack thread with VFS team](https://dsva.slack.com/archives/C04S98PE8FK/p1723653541711389)
- [x] Meeting date/time: **Monday, August 19, 2024 at 3:30pm ET**
  
##### Governance Team actions

- [x] Update this ticket with the Zoom recording
  - [Design intent meeting recording](https://oddball-io.zoom.us/rec/share/Y7v7sGPedL1cTRw8O5tBjJD9qIiMl8pCes7mhkLeBWgvhKpthe-4IZWm-ESNpIl5.ClHsDtdixrlJ03LG)
  - Passcode: `KLA3.n7^`
- Accessibility
  - [x] Feedback added to milestone
  - [ ] No feedback
- Design
  - [x] Feedback added to milestone
  - [ ] No feedback
- IA
  - [x] Feedback added to milestone
  - [ ] No feedback
  
##### VFS team actions

- [ ] Review feedback tickets. Comment on the ticket if there are any questions or concerns.
  
</details>

### Research Review
<details>
  <summary>Toggle Research Review</summary>

#### VFS actions
- [ ] Complete the [research review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Research-plan-review.1781891143.html)

</details>

### Midpoint Review
<details>
  <summary>Toggle Midpoint Review</summary>
  
##### VFS actions
Navigate to reference link: [Midpoint Review Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review)  
- [x] Schedule your Midpoint Review when ready:
   - Open the [Calendly midpoint review calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
   - Select a date and time and click “Confirm”
   - Add your name and email
   - Click "Add Guests" and enter the VFS meeting attendees email addresses
      - Invite all relevant VFS team members, including accessibility support, product owners, and other VA stakeholders
   - Click "Schedule Event"
- [ ] Check this box if you'd like this review to be asynchronous (Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review)
- [ ] Link all artifacts **ONLY** in the Midpoint Review artifacts section below at least two days before the scheduled Midpoint Review.  **Do NOT add artifacts to Comments section** 

**Midpoint Review artifacts**

See Platform guidance on [Midpoint Review artifacts](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review#Midpointreview-Artifacts). Platform feedback is based solely on the artifacts provided, as reviewed during the two days before the Midpoint Review meeting. Any work not included in the artifacts below or any ongoing work taking place during the review period may not be reflected in that feedback.

**Required artifacts**
- [x] Design prototypes
   - [ ] Our focus for MPR is on "first step"
      - [x] ["first step" desktop in Figma](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=613-299922&t=s1mW3znqpQRL8UVA-0)
      - [x] ["first step" mobile in Figma](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=618-308468&t=s1mW3znqpQRL8UVA-0)
   - [ ] We'd love to have async feedback on "more aggressive" option, see [video walk-through](https://www.loom.com/share/04be591bccae4ec9a9eebf3abaa70cee?sid=7d2598cf-b06c-41cd-9cb0-5c1ae19f00b8)
      - [x] ["more aggressive" desktop in Figma](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=491-208769&t=s1mW3znqpQRL8UVA-0)
      - [x] ["more aggressive" mobile in Figma](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=494-225703&t=s1mW3znqpQRL8UVA-0)
  - Note:
     -  "First step, less aggressive" we'll plan to take live ASAP, this is the one we want to have solid now, and ready for research.
     - "More aggressive option" we'll want ready in case we need it.
- [x] [Up to date user flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1724333905063/6c6192d24519142a244970b2db83539d4c2b1b2e?sender=u2bba352d5b2fc0f0969e0146)
  - [Guidance on user flows](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/standards/user-flow-guidance.md)
- [x] If working with [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request), [link to CAIA intake ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90690).

**Not required, but nice to have artifacts**
- [x] Accessibility annotations included as part of your design
- [x] [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Research/2024-08%20Sign%20in%20transition/2024-08%20Sign%20in%20transition%20research%20plan.md)
- Links to specific CAIA artifacts:
  - [ ] IA spec
  - [ ] Content source of truth
  - [x] [link to CAIA content suggestions in Figma](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/USiP-and-sign-in-modal---deemphasize-DSL%2FMHV?node-id=513-291491&t=FW7ho38X5H8C4kLZ-0)
- Any other artifacts you have so far
##### Platform actions
- [ ] [Slack thread with VFS team](https://dsva.slack.com/archives/C068QM7CCKG/p1724325740045879)
- [ ] Meeting date/time: 03:30pm - Tuesday, August 27, 2024 

##### Platform actions

* Accessibility
  * [ ] Feedback added to milestone
  * [x] No feedback
* Content
  * [x] Feedback added to milestone
  * [ ] No feedback
* Design
  * [x] Feedback added to milestone
  * [ ] No feedback
* IA
  * [x] Feedback added to milestone
  * [ ] No feedback
* [x] Update this ticket with the Zoom recording
  - [Recording link](https://oddball-io.zoom.us/rec/share/z6IPKQISHVUP38z8b8a10xz_-ALrbJ8-BQVboNbra3ah_xumSTrv0HL9KEQ4GaI5.A539XKESVYoy5yMQ
)
  - Passcode: ` vZ4ih.*2 `
  
##### VFS actions
  
- [ ] Review feedback tickets and comment on the ticket if there are any questions or concerns
  
</details>

### Contact Center Review
<details>
  <summary>Toggle Contact Center Review</summary>

#### VFS actions
- [x] Complete the [Contact Center review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html)
- [x] Link to the Contact Center review request ticket below
   - [Contact Center intake ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/93754)

</details>

### Staging Review
<details>
  <summary>Toggle Staging Review</summary>
  
##### VFS actions
- Navigate to reference link: [Staging Review Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/staging-review)  
- [x] Schedule your Staging Review when ready:
   - Open the [Calendly staging review calendar](https://calendly.com/collaboration-cycle/staging-review)
   - Select a date and time and click “Confirm”
   - Add your name and email
   - Click "Add Guests" and enter the VFS meeting attendees email addresses
      - Invite all relevant VFS team members, including accessibility support, product owners, and other VA stakeholders
   - Click "Schedule Event"
- [ ] If this product contains any [experimental design](https://design.va.gov/experimental-design/), add the `experimental-design` label and schedule a meeting with DSC to present the research findings.
- [ ] Link all artifacts **ONLY** in the Staging Review artifacts section below at least four days before the scheduled Staging Review.  **Do NOT add artifacts to Comments section**
- [x] I confirm the environment is available and test users have been provided.
- [x] Please verify your product information in the [Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory).
   - There is some information that needs to be updated (change in PM)
 
**Staging Review artifacts**

Links to [Staging Review artifacts](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Staging-review.1810137181.html#Stagingreview-Artifacts) must be added to this ticket 4 business days ahead of the scheduled meeting. Please do not make changes to the product or artifacts during the 4-day review period. 

#### Required artifacts
  - [x] **URL(s) to review the product**
    - The product should be available on an [approved staging environment](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/staging-environment-guidance-for-vfs-teams). 
    - List pages, sections of pages, and/or user flows impacted by this work.
       - [sign in modal](https://staging.va.gov/?next=loginModal&oauth=true)
       - [sign in page](https://staging.va.gov/sign-in/?oauth=true)
  - [x] **Drupal or Staging URL for updated primary entry point**: the main way Veterans will access the tool through site navigation (not search)
     - The primary entry point is simply VA.gov with many secondary entry points from various form and tool pages, and then a long tail of any page with the main VA nav header (which includes the "sign in" button)
  - [x] **Test users and scenarios (when applicable)**
    - Store [test user information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-accessible-example.md), passwords, and tasks in a `.md` file in the va.gov-team-sensitive repository. 
    - Make sure all user scenarios can be tested, i.e.: in-progress form, submitted form, new form.
    - N/A - sign in not needed to view these changes
  - [x] **Link to Sitewide CAIA intake ticket, if applicable.**
    - [[CAIA Intake] OCTO Identity : Sign in Transition #90690](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90690)
  - [x] **Completed [accessibility testing artifact](https://github.com/department-of-veterans-affairs/vets-website/pull/32164)**: [see instructions and link to accessibility testing template](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review).
  - [x] **QA Artifacts**: artifacts that correspond to each of the [QA Standards](https://depo-platform-documentation.scrollhelp.site/developer-docs/quality-assurance-standards).
    - [x] [Regression test plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Products/Secure%20Sign-in%20Transition/regression-test-plan.md)
    - [x] [Test plan](https://docs.google.com/spreadsheets/d/1g_CM9I0orpXWikR-BJPSKH3grgKS9VbRRMejCADZtDg/edit?usp=sharing)
    - [ ] Coverage for References
    - [ ] Summary (Defects) reports
    - [x] [E2E tests](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/login/tests/sign-in-page.cypress.spec.js)
    - [x] Code coverage (see below)
    - [x] [Endpoint monitoring playbook](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity/Troubleshooting_logging/Monitoring%20and%20Alerting/Monitors_Alerts_Datadog.md)

<summary>
Unit test code coverage
<details>
<img src="https://github.com/user-attachments/assets/c088909b-767d-40ec-986a-cb986ffc93b8" />

The Frontend Support Dashboard for [Unit Test Coverage Report](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/frontend-support-dashboard/unit-test-coverage-report) is inaccurate for the `login` application.

</details>
</summary>

#### Not required, but nice to have artifacts
  - [ ] **Content source of truth**: link to CAIA Content feedback, such as a content source of truth.
     - [CAIA ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90690)
     - [Figma design with finalized language](https://www.figma.com/design/skWgD0gHYGlKSoLdH097OX/Sign-in-changes-for-CSP-deprecation?node-id=1649-508314&node-type=frame&t=H2mfip9c9NHPhUJn-0) as suggested/reviewed by CAIA (and Collab Cycle) content folks and taken through usability sessions
  - [ ] **Information Architecture spec**: link to CAIA IA feedback, such as an IA spec.
 
##### Platform actions

- [x] [Slack thread with VFS team](https://dsva.slack.com/archives/C068QM7CCKG/p1727907773271199)
- [x] Meeting Date/Time: 11:30am - Tuesday, October 8, 2024 (Eastern Time - US & Canada)
- [ ] Update this ticket with the Zoom recording
  - Recording link: https://oddball-io.zoom.us/rec/share/WK4U7bjd0Ngeu-F7TKzkG6y0_YQWmPxJo9hlheYG2xfNJ7mTu6ekCF1eWZMisnGw.uMqbwPsgh9NqLku2
  - Password: 1#g3J23B
  
##### VFS actions
  
- [ ] Review the findings tickets and comment on the ticket if there are any questions or concerns
- [ ] Close individual findings tickets when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment on the issue ticket.
- [ ] After launch, [request an accessibility audit from the VA 508 Office](https://depo-platform-documentation.scrollhelp.site/developer-docs/request-support-from-the-va-508-office#RequestsupportfromtheVA508office-AuditRequest). This is required even if no accessibility issues were found during the Staging Review.
  - [ ] Share ServiceNow ticket number here: ______
- [ ] Close ticket once Privacy, Security, Infrastructure Readiness Review has been completed, VA 508 Office audit is requested, and all other post-Staging actions are complete
</details>

### Privacy, Security, Infrastructure Readiness Review
<details>
  <summary>Toggle Privacy, Security, Infrastructure Readiness Review</summary>
  
#### VFS actions
- [ ] Complete the [Privacy, security, infrastructure readiness review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Privacy-and-security-review.1782317101.html)
- [ ] Link to the Readiness Review ticket below

#### Platform actions
- [ ] Privacy, security, infrastructure readiness review is complete

</details>

**Comment by DanielleThierryUSDSVA** [comment]
Hi collab cycle team! I'm dropping in a note on behalf of @Samara-Strauss and myself to let you know that we're going to be starting with a less aggressive approach to the changes on the sign-in page. For conversation in design intent, but just wanted to make sure there's general awareness.

**Comment by Samara-Strauss** [comment]
To add to what @DanielleThierryUSDSVA said -- **yes**, we are starting with a less aggressive approach. However, I believe we are reviewing both the less aggressive approach to be implemented ASAP and a more aggressive approach that is being targeted for much later in the transition. I just wanted folks to be aware we are reviewing a couple different designs, but they are intended for different points in the transition. I will be on the call on Monday to help support the conversation.
